### PR TITLE
ENH: Add wrappings for video IO

### PR DESCRIPTION
**Pull request by tbirdso**
Adding Python wrappings for `itkVideoFileReader`, `itkVideoFileWriter`, and `itkFileListVideoIO`.

## PR Checklist
- [x] Added Python wrapping to new files (if any) as described in [ITK Software Guide](https://itk.org/ItkSoftwareGuide.pdf) Section 9.5

Refer to the [ITK Software Guide](https://itk.org/ItkSoftwareGuide.pdf) for
further development details if necessary.

### CODE REVIEW

File: Modules/Video/IO/wrapping/itkVideoFileReader.wrap [hunk 1]
```diff
@@ -0,0 +1,13 @@
+itk_wrap_include("itkImage.h")
+itk_wrap_include("itkVideoStream.h")
+
+itk_wrap_class("itk::VideoFileReader" POINTER)
+  UNIQUE(image_types "UC;ULL;D;${WRAP_ITK_ALL_TYPES}")
```
Comment (dzenanz) [inline Modules/Video/IO/wrapping/itkVideoFileReader.wrap:5]: Why introduce indentation here? Maybe just add an empty line before.

**Review by dzenanz — APPROVED** [review]

File: Modules/Video/IO/wrapping/itkVideoFileReader.wrap [hunk 2]
```diff
@@ -0,0 +1,12 @@
+itk_wrap_include("itkImage.h")
+itk_wrap_include("itkVideoStream.h")
+
+UNIQUE(image_types "UC;ULL;D;${WRAP_ITK_ALL_TYPES}")
+itk_wrap_class("itk::VideoFileReader" POINTER)
+  foreach(t ${image_types})
```
Comment (tbirdso) [inline Modules/Video/IO/wrapping/itkVideoFileReader.wrap:6]: @dzenanz Is this style what you mean?

Comment (dzenanz) [reply Modules/Video/IO/wrapping/itkVideoFileReader.wrap:6]: I thought  that two classes are wrapped here, `VideoFileReader` and `VideoStream`. So I meant:
```cmake

# wrap first class
itk_wrap_class("itk::VideoFileReader" POINTER)

# now wrap second class
UNIQUE(image_types "UC;ULL;D;${WRAP_ITK_ALL_TYPES}")
foreach(t ${image_types})
  # indentation comes inside a for loop
  ...
  foreach(...)
    # another indentation level
    ...
    # itk_wrap_template for itk::VideoStream
  endforeach()
endforeach()
```
now I see that `VideoFileReader` is wrapped for template parameter `VideoStream`, so my suggestion can be disregarded.

**Review by dzenanz — APPROVED** [review]

File: Modules/Video/IO/wrapping/itkVideoFileReader.wrap (hunk 2 above)
Comment (tbirdso) [reply Modules/Video/IO/wrapping/itkVideoFileReader.wrap:6]: Perfect, no problem!

**Review by thewtex — CHANGES_REQUESTED** [review]
We also want / need `itk::VideoIOBase` wrapped.

**Review by dzenanz — APPROVED** [review]

**Comment by dzenanz** [comment]
Maybe also rebase now?

**Review by thewtex — APPROVED** [review]
:+1: :100:

**Comment by thewtex** [comment]
CC @brad-t-moore @aylward

**Pull request merged**